sources.go

```diff
@@ -113,3 +113,19 @@ func Slice(s []string) *Pipe {
 func Stdin() *Pipe {
 	return NewPipe().WithReader(os.Stdin)
 }
+
+// Prompt presents a message to the user and returns a pipe containing user input or defaultValue provided.
+func Prompt(message, defaultValue string) *Pipe {
+
```

Comment:
It's a house style thing in this project that we don't use any blank lines within functions.

Reply:
Removed blank lines